ui/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "codegen": "openapi-rq -i http://localhost:8080/swagger-ui/api.json -o src/api"
+    "codegen": "openapi-rq -i http://localhost:${ORT_SERVER_CORE_PORT:-8080}/swagger-ui/api.json -o src/api"
```

Comment:
While there's probably no harm in this, IMO we should (and will need for deployment) set up an environment variable for the API URL, and use it as described [here](https://github.com/7nohe/openapi-react-query-codegen/issues/94#issuecomment-2079344928). Then the effect of this can be achieved by setting the `ORT_SERVER_CORE_URL` etc. with `http://localhost:8080` or `http://localhost:8081.`